sdk/resourcemanager/azure-resourcemanager-compute/src/main/java/com/azure/resourcemanager/compute/models/VirtualMachineScaleSet.java

```diff
@@ -1771,6 +1795,23 @@ WithSystemAssignedIdentityBasedAccessOrCreate withSystemAssignedIdentityBasedAcc
 
     /** Grouping of virtual machine scale set update stages. */
     interface UpdateStages {
+
+        /**
+         * The stage of a virtual machine scale set update allowing to specify the virtual network subnet for the
+         * primary network configuration.
+         */
+        interface WithNetworkSubnet {
+            /**
+             * Associate an existing virtual network subnet with the primary network interface of the virtual machines
+             * in the scale set.
+             *
+             * @param network an existing virtual network
+             * @param subnetName the subnet name
+             * @return the next stage of the definition
+             */
+            WithPrimaryLoadBalancer withExistingPrimaryNetworkSubnet(Network network, String subnetName);
```

Comment:
Where is this used/tested?

Reply:
It's used in update mode, I forgot to delete it. Thanks for pointing out.